src/main/java/seedu/address/logic/commands/ReviewCommand.java

```diff
@@ -60,6 +63,11 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Eatery eateryToAddReview = lastShownList.get(index.getZeroBased());
+
+        if (!eateryToAddReview.getIsOpen()) {
+            throw new CommandException(Messages.MESSAGE_EATERY_CLOSED);
```

Comment:
ah okay i see what you're trying to do now LMAO i think you can consider making this a static variable in this class instead of a static variable in Messages class because i feel like only this class would be calling this variable?

Reply:
Initially I thought so too but if other commands need to use it then need to change. There is a very high possibility of another command needing it so I just put it here

Reply:
ah i still think you should remove it from the Message class though as the Message class really seems like it's supposed to be for general application messages, and not messages that are specific to a particular set of commands?